apps/demo-bun-fuels/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "private": true,
+  "name": "demo-bun-fuels",
+  "description": "Simple demo using Bun with Fuels CLI",
+  "author": "Fuel Labs <contact@fuel.sh> (https://fuel.network/)",
+  "scripts": {
+    "pretest": "bun fuels build"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@fuel-ts/account": "workspace:^",
+    "@fuel-ts/errors": "workspace:^",
```

Comment:
I think we ship `errors` with `fuels` [now](https://github.com/FuelLabs/fuels-ts/pull/2398)?

Reply:
We do indeed! 🔍 🕵️

Reply:
Right. 👀 
 - https://github.com/FuelLabs/fuels-ts/issues/2223